test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaAttributeGenerator.java

```diff
@@ -57,6 +57,36 @@ public static Class<?> generateValidClassWithImplicitCreationAttribute(String na
 		return generator.defineClass(name, bytes, 0, bytes.length);
 	}
 
+	public static Class<?> generateFieldWithMultipleNullRestrictedAttributes(String className, String fieldClassName) throws Throwable {
+		/* Generate field class - value class with ImplicitCreation attribute and ACC_DEFAULT flag set  */
+		byte[] fieldClassBytes = generateClass(fieldClassName, ACC_FINAL + ValhallaUtils.ACC_VALUE_TYPE,
+			new Attribute[] {new ImplicitCreationAttribute(ValhallaUtils.ACC_DEFAULT)});
+		Class<?> fieldClass = generator.defineClass(fieldClassName, fieldClassBytes, 0, fieldClassBytes.length);
+
+		/* Generate class with field and multiple NullRestricted attributes */
+		byte[] classBytes = generateClassWithField(className, ValhallaUtils.ACC_IDENTITY,
+			"field", "L" + fieldClassName + ";", new Attribute[]{new NullRestrictedAttribute(), new NullRestrictedAttribute()});
```

Comment:
Instead of concatenating the descriptor, can you use `fieldClass.descriptorString()` to get that ?